templates/memcached_svcprop.erb

```diff
@@ -27,9 +27,9 @@ if @use_tls
   if @tls_ca_cert
     result << '"-o" "ssl_ca_cert="' + @tls_ca_cert + '"'
   end
-  result << '"-o" "ssl_verify_mode="' + @tls_verify_mode + '"'
+  result << '"-o" "ssl_verify_mode="' + @tls_verify_mode.to_s + '"'
   if @notls_listener_port
-    result << '"-l" "notls:"' + @notls_listener_addr + ':' + @notls_listener_port + '"'
+    result << '"-l" "notls:"' + @notls_listener_addr + ':' + @notls_listener_port.to_s + '"'
```

Comment:
Could this follow the convention of how real_listen_ip works?

real_listen_ip is a list (see line 45)

Reply:
@xek, I'm providing the argument in a different -l like this example:

`memcached -vvv -l 127.0.0.1 -p 11212 -Z -o ssl_verify_mode=1 -o ssl_chain_cert=tls/server_chain.pem -o ssl_key=tls/server.key -o ssl_ca_cert=tls/client-ca-root.crt -o ssl_verify_mode=1 -l notls:127.0.0.1:11211`

Reply:
@moisesguimaraes I think it makes sense to have a list here.
Something like
```
notls_listen => ['1.2.3.4:12345', '4.3.2.1:54321']
```
or even in the same way as it's done for listen_ip:
```
notls_listener_addr => ['1.2.3.4', '5.4.3.2']
```
and joining this list how it's done in https://github.com/saz/puppet-memcached/blob/39d052bed3a5f07889ba6a19398ec3c1fa344adb/templates/memcached_windows.erb#L10

Might also be a separate PR